crates/miden-tx/src/tests/kernel_tests/test_fpi.rs

```diff
@@ -706,42 +727,48 @@ fn test_simple_nested_fpi() {
             # push the index of desired storage item
             push.0
```

Comment:
We're passing this argument to `second_account_foreign_proc` but I don't think we're doing anything with it?